widget/embedded/src/components/BlockchainList/BlockchainList.styles.ts

```diff
@@ -1,9 +1,14 @@
 import { styled } from '@rango-dev/ui';
 
 export const List = styled('ul', {
+  height: 355,
```

Comment:
You are adding a fixed height to a general component. I don't think that's a good idea.